### DISCUSSION TITLE
Harbor containers using 10000 and tandp user and group permissions

### DISCUSSION THREAD

**Discussion by swasthibp** [goharbor/harbor#20519] [Q&A]
Hi team,

We are trying harbor installation which fails with all other users(we tried root) but works only with userid 10000  with below error:
08:27:29 [emerg] 1#0: cannot load certificate "/etc/cert/server.crt": BIO_new_file() failed (SSL: error:8000000D:system library::Permission denied:calling fopen(/etc/cert/server.crt, r) error:10080002:BIO routines::system lib)
nginx: [emerg] cannot load certificate "/etc/cert/server.crt": BIO_new_file() failed (SSL: error:8000000D:system library::Permission denied:calling fopen(/etc/cert/server.crt, r) error:10080002:BIO routines::system lib)

Can we please know what are these user and group dependency for harbor pods?

**Comment by stonezdj** [comment]
the /etc/cert/ is mounted from /data dir., all directories and files under /data directory should have read/write permission for 10000 user and group.

**Reply by swasthibp** [reply]
so this user cannot be modified?

**Reply by swasthibp** [reply]
Also what about the group to be used? Is that also specific?

**Reply by MinerYang** [reply]
Correct, user and group is specified in the image

**Reply by swasthibp** [reply]
could you please help me know where the user and group are specified in the image 
and also what is the default user and group?

**Reply by swasthibp** [reply]
I see that the default user and group are 10000, are these permissions configurable? If yes, are there any impacts?

**Reply by swasthibp** [reply]
Hi,

Any updates here please?
